action.yml

```diff
@@ -23,9 +23,11 @@ inputs:
   pre-test-cmd:
     description: Extra command to invoke before ansible-test
     default:
-  python-version:
-    description: Controller Python version
-    default: 3.9
+  env-python-version:
```

Comment:
Re: name

Maybe call this a worker/job Python version?

Reply:
`job-python-version` sounds good to me.

Reply:
Implemented in 277c5ef7c3d95604d6c9f74caee29c792c48ed60.